libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -56,7 +56,7 @@ CodeTransform::CodeTransform(
 	ExternalIdentifierAccess::CodeGenerator _identifierAccessCodeGen,
 	UseNamedLabels _useNamedLabelsForFunctions,
 	std::shared_ptr<Context> _context,
-	std::vector<TypedName> _delayedReturnVariables,
+	std::vector<NameWithDebugData> _delayedReturnVariables,
```

Comment:
Maybe it's better to use `NameWithDebugDataList` everywhere instead of `std::vector<NameWithDebugData>`.